tests/functional/api/test_merge_requests.py

```diff
@@ -163,28 +169,43 @@ def test_project_merge_request_approval_rules(group, project):
     assert len(approval_rules) == 1
     assert approval_rules[0].approvals_required == 2
 
-    approval_rules[0].delete()
-    ars = project.approvalrules.list(get_all=True)
-    assert len(ars) == 0
+    # Raises:
+    #   GitlabDeleteError: If the server cannot perform the request (the return code is not 2xx)
+    try:
+        approval_rules[0].delete()
+    except GitlabDeleteError as error:
+        assert False, f"approval_rules.delete raise error {error}"
+
 
+def test_merge_request_reset_approvals(gitlab_url, project):
+    today = date.today()
+    future_date = today + timedelta(days=4)
+    bot = project.access_tokens.create(
+        {"name": "bot", "scopes": ["api"], "expires_at": future_date.isoformat()}
+    )
+    logging.info("Created token for Bot")
 
-def test_merge_request_reset_approvals(gitlab_url, project, wait_for_sidekiq):
-    bot = project.access_tokens.create({"name": "bot", "scopes": ["api"]})
     bot_gitlab = gitlab.Gitlab(gitlab_url, private_token=bot.token)
+    logging.info("Created bot")
```

Comment:
I think we can delete the logs now if they were mostly used for debugging - same for any other logging added inside the actual test cases.

```suggestion
```

Reply:
Will go through my changelist and reduce logging and any other unnecessary changes